Client/mods/deathmatch/logic/CVehicleUpgrades.cpp

```diff
@@ -50,314 +52,18 @@ bool CVehicleUpgrades::IsUpgrade(unsigned short usModel)
 bool CVehicleUpgrades::IsUpgradeCompatible(unsigned short usUpgrade)
 {
     unsigned short     us = usUpgrade;
```

Comment:
I would remove `us` and use `usUpgrade` instead.

Reply:
I must have missed that, thanks. Done.